run_generate.py

```diff
@@ -13,10 +13,22 @@
 from pathlib import Path
 
 import torch
+
 from nanotron import distributed as dist
 from nanotron import logging
-from nanotron.config import GenerationArgs, LoggingArgs, ParallelismArgs, get_config_from_file
-from nanotron.generation.decode import GenerationInput, TokenizerConfig, decode_text, decode_tokenized
+from nanotron.config import (
```

Comment:
use linter as [here](https://github.com/huggingface/nanotron?tab=readme-ov-file#installation)

Reply:
should be good for now